README.md

```diff
@@ -1,77 +1,150 @@
-| `trigger_2_labels` | array | **Required** | The class names from `trigger_2_detector` that count as valid. |
-| `trigger_2_confidence` | float | Optional | The detection confidence needed in order to move into the `COUNTDOWN` state. Default value is 0.5. |
-| `verification_detector` | string | **Required** | The name of the vision service detector that will be used to verify the object. |
-| `verification_labels` | array | **Required** | The class names from `verification_detector` that count as valid. |
-| `verification_confidence` | float | Optional | The detection confidence needed in order to move into the `DISARMED` state. Default value is 0.8 |
-| `countdown_time_s` | int | Optional | The time in seconds the system will remain in state `COUNTDOWN` before transitioning to state `ALARM`. Default value is 20. |
-| `alarm_time_s` | int | Optional | The time in seconds the system will remain in  state `ALARM` before transitioning to state `TRIGGER_1`. Default value is 10. |
-| `disarmed_time_s` | int | Optional | The time in seconds the system will remain in  state `DISARMED` before transitioning to state `TRIGGER_1`. Default value is 10. |
-| `disable_alarm` | bool | Optional | Disables the `COUNTDOWN` and `ALARM` states. The system will always remain the `TRIGGER_1` and `TRIGGER_2` states. Default value is False. |
+
+#### Attributes
+
+| Name  | Type | Inclusion | Description |
+| ------------- | ------------- | ------------- | ------------- |
+| `recognition_model` | string | **Required** | The model to use for facial recognition.<br/>`ArcFace` is set as the default. |
+| `detection_framework`  | string | **Required** | The detection framework to use for facial detection.<br/>`ssd` is set as the default. |
+| `face_labels` | dictionary[string,string] | Optional | The dictionary representing labels for photos of individuals faces paired with the image path on your machine running `viam-server`. |
+| `verify_threshold` | float64 | Optional | The confidence threshold for face verification. <br/>Only applicable if `disable_verify` is `false` and `face_labels` are set. Faces below this threshold will be labeled as normal detected `"face"`. <br/>Defaults to `0.8`. |
+| `disable_detect` | bool | Optional | If set to `true`, facial detection will be disabled.<br/>Only applicable if `disable_verify` is `false` and `face_labels` are set. Any faces detected without being verified as matching a label will be labeled as `"face"`.<br/>Defaults to `false`. |
+| `disable_verify` | bool | Optional | If set to `true`, facial verification will be disabled. <br/>Defaults to `false`. <br/>Only applicable if `face_labels` are set. If you only want verified faces to be labeled and skip labeling unverified visitors as `"face"`, set this to `false` and `disable_detect` to `true`. |
+
+> [!NOTE]
+> For more information on the available attributes, see the [`facial-detector` module documentation](https://github.com/viam-labs/facial-detection).
+
```

Comment:
Add build and run section here following template: https://github.com/viam-labs/generator-viam-module/blob/main/generators/app/templates/python/README.md#build-and-run